server/events/project_command_pool_executor.go

```diff
@@ -81,6 +81,9 @@ func runProjectCmdsParallelGroups(
 	for _, group := range groups {
 		res := runProjectCmdsParallel(group, runnerFunc, poolSize)
 		results = append(results, res.ProjectResults...)
+		if res.HasErrors() {
```

Comment:
I can add a flag if you want that. Should that be added on a repo level? In the issue that I referred to, the suggestion was setting this as the expected behaviour. 

I'm not sure I see when you would want to opt out of this as you want it to run in a given order, but I'm all ears for what you guys want 👍

Reply:
I'm not an official maintainer so my opinion is lower than what @krrrr38 mentioned in the issue 😄 
It may not be worth it but my thought process was that it changes existing behavior so if any users _don't_ want this it would break their workflow.

Reply:
I agree with @jukie , repo level will be nicer and allows the user to change the behaviour by a flag

Reply:
Why would anyone want their execution group to continue if it has errors in between?